<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowLive</class>
 <widget class="QMainWindow" name="MainWindowLive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkApplyAlgorithms">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkUpdateZ">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonUpdateZ">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRenewReference">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelFPS">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spinRefreshFPS">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="SpectrogramPlot" name="graphSpectrogram" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="Surface3DPlot" name="graph3DSurface" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="ParametricSurface3DPlot" name="graph3DParametric" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelThreshold">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editThreshold">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="Graph1D" name="graphSection" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSpectrogram"/>
   <addaction name="action3dSurface"/>
   <addaction name="actionParametric"/>
   <addaction name="actionLine"/>
  </widget>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/mainwindowlive/play.png</normaloff>:/icons/icons/mainwindowlive/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/mainwindowlive/record.png</normaloff>:/icons/icons/mainwindowlive/record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/mainwindowlive/stop.png</normaloff>:/icons/icons/mainwindowlive/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSpectrogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/mainwindowlive/spectrogram.png</normaloff>:/icons/icons/mainwindowlive/spectrogram.png</iconset>
   </property>
   <property name="text">
    <string>Spectrogram</string>
   </property>
  </action>
  <action name="action3dSurface">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/mainwindowlive/surfaceplot.png</normaloff>:/icons/icons/mainwindowlive/surfaceplot.png</iconset>
   </property>
   <property name="text">
    <string>3d Surface</string>
   </property>
  </action>
  <action name="actionParametric">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/mainwindowlive/parametricplot.png</normaloff>:/icons/icons/mainwindowlive/parametricplot.png</iconset>
   </property>
   <property name="text">
    <string>Parametric</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/icons/mainwindowlive/lineplot.png</normaloff>:/icons/icons/mainwindowlive/lineplot.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpectrogramPlot</class>
   <extends>QWidget</extends>
   <header>spectrogramplot.h</header>
  </customwidget>
  <customwidget>
   <class>Surface3DPlot</class>
   <extends>QWidget</extends>
   <header>surface3dplot.h</header>
  </customwidget>
  <customwidget>
   <class>ParametricSurface3DPlot</class>
   <extends>QWidget</extends>
   <header>parametricsurface3dplot.h</header>
  </customwidget>
  <customwidget>
   <class>PlotData1D</class>
   <extends>QWidget</extends>
   <header>plotdata1d.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
